FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -266,7 +265,16 @@ extension MessageViewController {
     }
 
     @objc private func handleMailTap() {
```

Comment:
Instead of `Mail` here we should call it `MarkUnread` so this would be `handleMarkUnreadTap`. If the icon is elsewhere called just `Mail` in the code, than that should be updated too.